docker/service/src/integration-test/java/uk/nhs/adaptors/scr/SpineMockServiceTest.java

```diff
@@ -51,4 +56,46 @@ public void sampleEndpointShouldReturnMockedData() {
 
         assertThat(dataFromSpine).isEqualTo(message);
     }
+
+    @Test
```

Comment:
I think that a good test would need two more steps:
1) Send HTTP request to SCR API instead of calling spineClient directly
2) Verification if request sent to Spine is correct

Reply:
I have kept this test in for fake spine 200 OK, and added additional class for acstest adding this example :)